src/ucp/wireup/wireup.c

```diff
@@ -717,6 +717,49 @@ static ucs_status_t ucp_wireup_resolve_proxy_lanes(ucp_ep_h ep)
     return UCS_OK;
 }
 
+void ucp_wireup_print_used_tls(ucp_ep_h ep, ucp_rsc_index_t aux_rsc_index)
+{
+    ucp_context_h context          = ep->worker->context;
+    const ucp_ep_config_key_t *key = &ucp_ep_config(ep)->key;
+    char info[256]                 = {0};
+    ucp_rsc_index_t tl_id;
+    ucp_lane_index_t lane;
+    char *p, *endp;
+    int used;
+
+    if (!ucs_log_is_enabled(UCS_LOG_LEVEL_INFO)) {
+        return;
+    }
+
+    p    = info;
+    endp = p + sizeof(info);
+
+    snprintf(p, endp - p,  "ep %p tls: ", ep);
+    p += strlen(p);
+
+    ucs_for_each_bit(tl_id, context->tl_bitmap) {
+        used = 0;
+        for (lane = 0; lane < key->num_lanes; ++lane) {
+            if (key->lanes[lane].rsc_index == tl_id) {
```

Comment:
can we use ucp_ep_config_get_multi_lane_prio() to search?

Reply:
here we check rsc indexes, not lanes. How could I reuse it?

Reply:
right